src/com/amazon/ion/benchmark/Main.java

```diff
@@ -313,9 +316,18 @@ public static void main(String[] args) {
         if (optionsMap.get("--help").equals(true)) {
             printHelpAndExit();
         }
+       
         try {
-            OptionsMatrixBase options = OptionsMatrixBase.from(optionsMap);
-            options.executeBenchmark();
+        	 if (optionsMap.get("generate").equals(true)) {
+        		 try {
+        				WriteRandomIonValues.writeRandomDecimals(Integer.valueOf(optionsMap.get("<size>").toString()),optionsMap.get("<file>").toString());
+        			} catch (Exception e) {
+        				e.printStackTrace();
+        			}
```

Comment:
There's no need to try/catch here. Any exception thrown can just be thrown normally from `main`.